source/contracts/reporting/ReportingWindow.sol

```diff
@@ -216,6 +217,29 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
         return true;
     }
 
+    // This exists as an edge case handler for when a ReportingWindow has no markets but we want to migrate fees to a new universe. If a market exists it should be migrated and that will trigger a fee migration. Otherwise calling this on the desitnation reporting window in the forked universe with the old reporting window as an argument will trigger a fee migration manaully
+    function triggerMigrateFeesDueToFork(IReportingWindow _reportingWindow) public afterInitialized returns (bool) {
+        require(_reportingWindow.getNumMarkets() == 0);
+        _reportingWindow.migrateFeesDueToFork();
+    }
+
+    function migrateFeesDueToFork() public afterInitialized returns (bool) {
+        // NOTE: Will need to figure out a way to transfer other denominations when that is implemented
+        ICash _cash = ICash(controller.lookup("Cash"));
+        uint256 _balance = _cash.balanceOf(this);
+        if (_balance == 0) {
+            return false;
+        }
+        IReportingWindow _shadyReportingWindow = IReportingWindow(msg.sender);
+        IUniverse _shadyUniverse = _shadyReportingWindow.getUniverse();
+        require(_shadyUniverse.isContainerForReportingWindow(_shadyReportingWindow));
+        require(universe.isParentOf(_shadyUniverse));
+        _cash.transfer(_shadyReportingWindow, _balance);
```

Comment:
Same comment as above, rename `_shady*` to something not shady as soon as we have finished validating to make it clear to the reader when validation is complete and the address can be trusted.